tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1296,6 +1296,16 @@ void libfmt_formatter_test_runtime_precision() {
     assert(format(STR("{0:.{1}}"), STR("str"), 2) == STR("st"));
 }
 
+template <class charT>
+void test_locale_speicifc_formatting_without_locale() {
+#if !defined(_DLL) || _ITERATOR_DEBUG_LEVEL == DEFAULT_IDL_SETTING
+    std::locale loc("en_US.UTF-8");

```

Comment:
I think this should be hyphenated (other tests say `"en-US"`, and we've had repeated issues with underscore locales on different OSes). Also, because it's UTF-8, I think we need to guard this to run for GitHub only, as the internal test machines are too old. I'll do so pre-emptively because discovering this during the port process will be a timesink and GitHub provides sufficient coverage.